etp-front/src/components/text-editor/quill.js

```diff
@@ -64,7 +74,7 @@ export const quill = (
   );
   remove.href = '';
 
-  const focusout = event => {
+  const focusout = () => {
```

Comment:
Now it seems to be using a global event, but could switch back to using the one from the parameters